static/js/place/util.ts

```diff
@@ -247,3 +248,47 @@ export function displayNameForPlaceType(
   }
   return intl.formatMessage(retMessage);
 }
+
+// Get ordered display topics give a set of chart configs.
+export function getDisplayTopics(
```

Comment:
wdyt of moving up this up to the server? since we are in control of the endpoints, we should be able to return the data in display order. can we add a todo? we'd have to do that along with splitting data for each page/topic.

Reply:
I made the change and ended up with a larger refactor and clean up. They are noted in the PR description